src/compiler/checker.ts

```diff
@@ -1100,7 +1098,7 @@ namespace ts {
         }
 
         function getDeclarationOfAliasSymbol(symbol: Symbol): Declaration | undefined {
-            return forEach(symbol.declarations, d => isAliasSymbolDeclaration(d) ? d : undefined);
+            return find<Declaration>(symbol.declarations, isAliasSymbolDeclaration);

```

Comment:
`symbol.declarations` is `Declaration[]` but `isAliasSymbolDeclaration` takes a `Node`. It's odd that this infers `Node` as the type. Should this be an issue?